build.assets/docker/base.dockerfile

```diff
@@ -81,13 +81,21 @@ RUN export DEBIAN_FRONTEND=noninteractive && set -ex \
         automake \
         pkg-config \
         libmnl-dev \
-        make \
-        && apt-get -y autoclean && apt-get -y clean && apt-get autoremove -y \
-        && rm -rf /var/lib/apt/lists/*;
+        make
+
+# Deploy Ubuntu .deb in order to get .socket activated iscsid. Debian's iscsid is not socket activated.
+# Socket activation is needed in order to avoid developing custom service start logic when OpenEBS app is deployed.
+RUN cd /var/lib/dpkg/
+RUN wget  http://mirrors.kernel.org/ubuntu/pool/main/o/open-iscsi/open-iscsi_2.0.874-7.1ubuntu6_amd64.deb
```

Comment:
Let's define the iscsi package version in the top Makefile and propagate it here like we do for other component versions.

Also, nit: spotted an extra space after "wget".